tools/clang/lib/SPIRV/LowerTypeVisitor.h

```diff
@@ -31,6 +31,7 @@ class LowerTypeVisitor : public Visitor {
   bool visit(SpirvModule *, Phase) override { return true; }
   bool visit(SpirvFunction *, Phase) override;
   bool visit(SpirvBasicBlock *, Phase) override { return true; }
+  //bool visit(SpirvLoad *, Phase) override;
```

Comment:
nit: if it is not used, could you please remove this line?

Reply:
Oops, that was something I forgot to remove before committing, will remove.